### PR TITLE
Added git submodule init and update

### PR DESCRIPTION
**Pull request by gyliu513**
### Description of your changes

Fixes https://github.com/crossplane-contrib/provider-kubernetes/issues/9

I have:

- [x] Read and followed Crossplane's [contribution process].
- [x] Run `make reviewable test` to ensure this PR is ready for review.

### How has this code been tested

[contribution process]: https://git.io/fj2m9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ spec:
 Start a local development environment with Kind where `crossplane` is installed:
 
 ```
+git submodule init
+git submodule update
```
Comment (turkenh) [inline README.md:37]: We actually have a make target (`fallthrough`) doing this which should be the first target in the makefile which is not the case currently. It is invoked if no target specified (e.g. just `make`)

So, we should better instruct to call `make` here and move `local-dev` target down in the Makefile as below:

```
diff --git a/Makefile b/Makefile
index 6cdd9a8..773d903 100644
--- a/Makefile
+++ b/Makefile
@@ -49,7 +49,6 @@ IMAGES = provider-kubernetes provider-kubernetes-controller
 # Setup Local Dev
 -include build/makelib/local.mk

-local-dev: local.up local.deploy.crossplane
 # ====================================================================================
 # Targets

@@ -82,6 +81,8 @@ generate.done: crds.clean
 # integration tests
 e2e.run: test-integration

+local-dev: local.up local.deploy.crossplane
+
 # Run integration tests.
 test-integration: $(KIND) $(KUBECTL) $(HELM3)
        @$(INFO) running integration tests using kind $(KIND_VERSION)
```

**Review by turkenh — COMMENTED** [review]
Thanks for your contribution @gyliu513!

Added a comment to fix this using the existing make target.

File: README.md (hunk 1 above)
Comment (gyliu513) [reply README.md]: Done, thanks @turkenh

**Review by turkenh — APPROVED** [review]
Thanks!

**Pull request merged**